docs/configuration.md

```diff
@@ -305,7 +305,7 @@ Apart from these, the following properties are also available, and may be useful
   <td>daily</td>
   <td>
     Set the time interval by which the executor logs will be rolled over.
-    Rolling is disabled by default. Valid values are `daily`, `hourly`, `minutely` or
+    Rolling is disabled by default. Valid values are <code>daily</code>, <code>hourly<code>, <code>minutely<code> or
```

Comment:
@jaceklaskowski The `<\code>` end tag was wrong instead of `<code>`. Maybe it was a typo, but it will lead this page chaos. I may PR to fix this later.
cc @srowen